kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/UIServicesClient.java

```diff
@@ -67,4 +69,11 @@
      * @return svg (xml) representing process image annotated with active (in red) and completed (in grey) nodes
      */
     String getProcessInstanceImage(String containerId, Long processInstanceId);
+
+    /**
+     * Override default response handler to change interaction pattern. Applies only to JMS
+     * based integration.
+     * @param responseHandler
```

Comment:
I believe we should not use empty `@params` in javadoc. I would either remove it or add some description.